source/gridfs/gridfs-spec.rst

```diff
@@ -59,8 +59,8 @@ Chunk
   The default size for the data field in chunks is 255 KiB. Chunk documents have the following form:
 
   .. code:: javascript
-  
-    { 
+
+    {
```

Comment:
Sorry for all the whitespace changes. The editor automatically removed all blanks at the end of a line (that's a good thing).